Microsoft/WebTemplateStudio
# angular symbol links are relative path only and 2 clean up

This change is to address "symbol imports aren't consistent" suggestion from #681. Also, did a little clean up:  removed an unused import and renamed a component file to end with ".component.ts"

**How to test:**
1. generate an angular project with at least 4 type of pages selected
2. you should be able to start the project (there should be no import errors)
